features/lwipstack/mbed_lib.json

```diff
@@ -72,6 +72,22 @@
             "help": "Maximum number of open UDPSocket instances allowed, including one used internally for DNS.  Each requires 84 bytes of pre-allocated RAM",
             "value": 4
         },
+        "memp-num-tcp-seg": {
+            "help": "Number of simultaneously queued TCP segments. Current default (used if null here) is set to 16 in opt.h, unless overridden by target Ethernet drivers.",
+            "value": null
```

Comment:
Maybe we could use default values here, instead of `null` to get rid of `#ifdef` in lwipopts.h

Reply:
RZ/A1 would like to use the customized value not default value(16) against this element for speeding up. Thus by setting this element as null , the other boards excepting RZ/A1 use default value. RZ/A1 use customized value for speeding up.